server/src/main/resources/openapi/reservoir.yaml

```diff
@@ -892,6 +892,173 @@ paths:
           $ref: "#/components/responses/trait_404"
         "500":
           $ref: "#/components/responses/trait_500"
+  /reservoir/solr:
+    parameters:
+      - $ref: headers/okapi-tenant.yaml
+      - $ref: headers/okapi-token.yaml
+      - $ref: headers/okapi-url.yaml
+    post:
+      description: Create Solr client
```

Comment:
How about we use a different name from "client"? It's a bit confusing name that one might understand to mean data fetching. We could use "exporter" or "publisher". Or something more abstract like "outlet" (to keep with the naming convention which would be my preference).

Reply:
Sure. So you prefer outlet. An plural of that is Solr outlets?

Reply:
Yep.

Reply:
Renamed now